third_party/sigs.k8s.io/controller-runtime/pkg/dynamiccache/cache.go

```diff
@@ -54,7 +54,7 @@ func defaultOpts(config *rest.Config, opts cache.Options) (cache.Options, error)
 	// Construct a new Mapper if unset
 	if opts.Mapper == nil {
 		var err error
-		opts.Mapper, err = apiutil.NewDynamicRESTMapper(config)
+		opts.Mapper, err = apiutil.NewDiscoveryRESTMapper(config)
```

Comment:
Was this change intentional? I think for dynamicCache, DynamicRESTMapper is the correct default.

Reply:
Wasn't intentional, got reverted in rebase. Fixed now, thanks